examples/Helpers/MonitorNumbersToFile.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Helpers;
+
+use TelegramOSINT\Scenario\PresenceMonitoringScenario\PresenceMonitoringCallbacks;
+
+class MonitorNumbersToFile implements PresenceMonitoringCallbacks{
+
+    private $lines = [];
+    private $statuses = [];
+    private $fileName;
+
+    public function __construct(string $fileName)
+    {
+        $this->fileName = $fileName;
+    }
+
+    public function onOnline(string $number)
+    {
+        $this->setupNumber($number, 1);
```

Comment:
Can we use constants for these statuses? I see they are used multiple times.

Reply:
++